eva-accession-pipeline/src/test/resources/accession-pipeline-test.properties

```diff
@@ -8,7 +8,7 @@ parameters.assemblyAccession=assembly
 parameters.taxonomyAccession=1111
 parameters.projectAccession=project
 parameters.chunkSize=100
-parameters.vcf=src/test/resources/input-files/vcf/genotyped.vcf.gz
+parameters.vcf=src/test/resources/input-files/vcf/small_genotyped.vcf.gz
```

Comment:
A header line indicating that one record is a non-variant would be useful, because it took me a while to understand why only 22 out of 23 were expected to be accessioned.